DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/RecPoints.h

```diff
@@ -10,80 +10,102 @@
 
 /// \file RecPoints.h
 /// \brief Definition of the FIT RecPoints class
-#ifndef ALICEO2_FIT_RECPOINTS_H
-#define ALICEO2_FIT_RECPOINTS_H
+#ifndef ALICEO2_FT0_RECPOINTS_H
+#define ALICEO2_FT0_RECPOINTS_H
 
 #include "CommonDataFormat/InteractionRecord.h"
 #include "CommonDataFormat/TimeStamp.h"
+#include "DataFormatsFT0/ChannelData.h"
+#include "CommonDataFormat/RangeReference.h"
+#include "DataFormatsFT0/Digit.h"
 #include <array>
 #include "Rtypes.h"
 #include <TObject.h>
-#include <DataFormatsFT0/Digit.h>
+#include <gsl/span>
 
 namespace o2
 {
 namespace ft0
 {
+struct ChannelDataFloat {
+
+  //public:
+
+  int ChId = -1;       //channel Id
```

Comment:
Do you really need doubles here (and again, what is the range of int members?) Anyway, if the types cannot be changed, please move ChainQTC right after the ChId, this will save 8 B in the sizeof(ChannelDataFloat)